audonnx/core/testing.py

```diff
@@ -0,0 +1,169 @@
+import typing
+
+import numpy as np
+import onnx
+
+import audeer
+import audobject
+import audonnx
+
+
+def create_model(
+        root: str,
+        shapes: typing.Sequence[typing.Sequence[int]],
+        *,
+        value: float = 0.,
+        dtype: int = onnx.TensorProto.FLOAT,
+        opset_version: int = 14,
+) -> audonnx.Model:
+    r"""Create test model.
+
+    Creates a model that outputs
+    arrays filled with ``value``
+    of the given ``shapes``.
+    For each entry an output node will be created.
+    ``-1``, ``None`` or strings
+    define a dynamic axis.
+    Per node,
+    one dynamic axis is allowed.
+    The model is stored under ``root``
+    and can be loaded with
+    :func:`audonnx.load`.
```

Comment:
This might be slightly misleading, as in the example you do:

```python
model = audonnx.testing.create_model('test', shapes)
```
and not
```python
audonnx.testing.create_model(root, shapes)
model = audonnx.load(root)
```

So maybe change to

```suggestion
    :func:`audonnx.load`,
    which is the default behavior.
```

Or maybe only mention that it is stored under `root`?

Reply:
I checked and it's possible to create a `InferenceSession` directly from an onnx graph. I.e. we can also create a model without storing the ONNX file, then we don't have to pass a `root ` folder. We only have to ensure that (when created directly from a graph) the ONNX file is saved when the user calls `to_yaml()`. If we think that would be a useful extension, we can work on this in another PR. For now, I simply removed the sentence.

Reply:
Good to know that this is also possible.

At the moment, we anyway have a always a folder as a model, so I think for a start the current implementation makes the most sense.